### PR TITLE
Feature/cli wrapper

### PR DESCRIPTION
**Pull request by CalebMuhia**

### CODE REVIEW

File: docker/manage_sdk.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+
+def get_image():
+    images = subprocess.check_output(
+        "docker images --format '{{.Repository}}' platerecognizer/alpr*".split(
+        )).decode().split('\n')
+    return images[0].replace("'", "")
+
+
+def verify_token(token, license_key):
```
Comment (marcbelmont) [inline docker/manage_sdk.py]: license_key validity is not checked

File: docker/manage_sdk.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+            cmd = 'docker run --rm -t -v license:/license -e TOKEN={} -e UNINSTALL=1 {}'.format(
+                token, image)
+            os.system(cmd)
+
+        elif uninstall_choice == 2:
+            stop_container(image)
+            cmd = 'docker run --rm -t -v license:/license -e TOKEN={} -e UNINSTALL=1 {}'.format(
+                token, image)
+            os.system(cmd)
+            # TODO: command is not complete on the docs
```
Comment (marcbelmont) [inline docker/manage_sdk.py]: Can you elaborate?

File: docker/manage_sdk.py [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+            image_version='latest'):
+    test_existing_install(image)
+    pull_cmd = 'docker pull {}:{}'.format(image, image_version)
+    os.system(pull_cmd)
+    run_cmd = '{} run {} {} -p {}:8080 -v license:/license -e TOKEN={} -e LICENSE_KEY={} {}'.format(
+        docker_version,
+        "-dit --restart unless-stopped" if auto_start_container else '-it --rm',
+        extra_args, port, token, license_key, image)
+    os.system(run_cmd)
+    if test_install(image):
```
Comment (marcbelmont) [inline docker/manage_sdk.py]: Make a call to the SDK endpoint to ensure it is running.

File: docker/manage_sdk.py (hunk 2 above)
Comment (CalebMuhia) [reply docker/manage_sdk.py]: it seems like the command on the sdk page is not complete, please take a look at this screenshot https://prnt.sc/qbn574

File: docker/manage_sdk.py (hunk 1 above)
Comment (CalebMuhia) [reply docker/manage_sdk.py]: we are sending a request to the `v1/sdk-webhooks/`, how should we verify the the `license key`?

File: docker/manage_sdk.py (hunk 2 above)
Comment (marcbelmont) [reply docker/manage_sdk.py]: can you put the correct command in manage_sdk.py?
would be something like docker rm image_id

File: docker/manage_sdk.py (hunk 1 above)
Comment (marcbelmont) [reply docker/manage_sdk.py]: I think status code is 200 if correct

Comment (marcbelmont) [reply docker/manage_sdk.py]: let's modify the api to return a 404 when the license_key can't be found.

Comment (marcbelmont) [reply docker/manage_sdk.py]: if you get a 403, the token is invalid. Display a specific error message if the token is wrong or the license is wrong.

File: docker/PlateRec_SDK_Manager.py [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+        if choice == '4':
+            print("Quit!")
+            exit(1)
+        if choice in ['1', '2', '3']:
+            action_choice = choice
+            break
+        else:
+            print('Incorrect Choice')
+
+    if action_choice == '1':
```
Comment (marcbelmont) [inline docker/PlateRec_SDK_Manager.py]: Function main is too long. Can you refactor and do the following instead?

```
if action_choice == '1':
  return install(...)
elif action_choice == '2':
  return update(...)
...
```

Comment (CalebMuhia) [reply docker/PlateRec_SDK_Manager.py]: done

**Pull request merged**